modules/src/what_cocktail.py

```diff
@@ -0,0 +1,351 @@
+#rewrite the program (which a burglar stole from me!) and then release it onto telegram and messenger!
+''' argh what cocktail can i make with these ingredients? search and return a cocktail which has
+the most matching ingredients'''
+from enchant.checker import SpellChecker
+import re
+
+print("What? Cocktail")
```

Comment:
This is more like a Python3 syntax for print.

Reply:
it is python3, is that an issue?

Reply:
Ahhhh I didn't realise!! hmm do you suggest rewriting for python 2 or is there the possibility of a python3 version coming out? or will it just work?

Reply:
also who do I update you with my file changes?

Reply:
I guess the update is something only @swapagarwal has a clue about. But stick to Py2 for the moment.